packages/m/mootable-hashmap.json

```
@@ -0,0 +1,34 @@
+{
+  "description": "Full featured, fast, HAMT based HashMaps",
+  "filename": "index.js",
```

```suggestion
  "filename": "hashmap.min.js",
```
It looks like the files are called hashmap, not index :)

---

sure is hashmap.min the correct one or just hashmap, I provide both? For now I will make it just hashmap.